doc/pages/commands.asciidoc

```diff
@@ -14,6 +14,12 @@ command *q!* has to be used). Aliases are mentionned below each commands.
 For the following *write* commands, the *-sync* switch forces the synchronization
 of the file onto the filesystem
 
+*arrange-buffers* <buffer>...::
+    reorder the buffers in the buffers list
+    the list will be sorted following the order specified in the parameters
```

Comment:
I worry that the word "sorted" will make people expect some kind of comparison function. How about:

> The named buffers will be moved to the front of the buffer list, in the order given